test_runner/regress/test_vm_truncate.py

```diff
@@ -0,0 +1,31 @@
+from fixtures.neon_fixtures import NeonEnv
+
+
+#
+# Test that VM is properly truncated
+#
+def test_vm_truncate(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+
+    endpoint = env.endpoints.create_start("main")
+    con = endpoint.connect()
+    cur = con.cursor()
+    cur.execute("CREATE EXTENSION neon_test_utils")
+    cur.execute("CREATE EXTENSION pageinspect")
+
+    cur.execute(
+        "create table t(pk integer primary key, counter integer default 0, filler text default repeat('?', 200))"
+    )
+    cur.execute("insert into t (pk) values (generate_series(1,1000))")
+    cur.execute("delete from t where pk>10")
+    cur.execute("vacuum t")  # rtruncate all forks pf reation
+    # get image of first block of VM updted by Postgres excluding page header
+    cur.execute("select substr(encode(get_raw_page('t', 'vm', 0), 'hex'), 48)")
```

Comment:
Why '48' ? Can the fields in the page header differ?

I remember we talked about the LSN, and you said that the LSN can differ, but I don't remember what the reason was. Please explain that in a comment. Can the other fields differ?

Reply:
24 - is size of page header.
48 is it's hexadecimal representation.
In this test  I just ignoring page header because we only want to be sure that content of bitmap is equal.
Actually in most cases page header is also not changed. But I do not think that it should be checked here.
I will add comment.